src/Hypergrid.js

```diff
@@ -1670,6 +1673,11 @@ Hypergrid.prototype = {
         return result;
     },
 
+    getHiddenColumns: function(){
+        //A non in-memory behavior will be more troublesome
+      return this.behavior.getHiddenColumns();
```

Comment:
I'm not sure if this should be exposed on the Grid object as remote behavior wouldn't want access to call ALL hidden columns